Tools/WebKitTestRunner/InjectedBundle/gtk/ActivateFontsGtk.cpp

```diff
@@ -38,14 +38,10 @@
 
 namespace WTR {
 
-void initializeFontConfigSetting()
+void activateFonts()
 {
-    if (g_getenv("WEBKIT_SKIP_WEBKITTESTRUNNER_FONTCONFIG_INITIALIZATION"))
-        return;
```

Comment:
Why removing this too?

Reply:
I didn't find any usage of this variable. As far as I understand, it could be useful in the past in the case where we don't have the test fonts for some reason. But now these fonts are always available, so I can't imagine any case when this variable is needed.

Reply:
Okay then, I'll revert this part.